grails-app/taglib/org/pih/warehouse/AuthTagLib.groovy

```diff
@@ -18,9 +19,24 @@ class AuthTagLib {
     def userService
 
     def supports = { attrs, body ->
-        def location = Location.load(session.warehouse.id)
-        if (location && location.supports(attrs.activityCode)) {
```

Comment:
What with other existing usages of `g:supports activityCode`? Looks like you removed the check for activityCode, but you did not modify all usages. I'm fine with unifying `authorize` with `supports`, but then you need to make sure everything was covered. I suggest leaving the chack for single `activityCode` too.

@jmiranda are you fine with unifying `authorize` with `supports` tags, since those two seem to do the same?

Reply:
I guess so?